src/SourcePathsMappingUI/AskUserForFile.cpp

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
```

Comment:
This and the next commit should probably be merged or it may cause linker errors due to the redefinition of TryAskingTheUserAndReadSourceFile, won't it?

Reply:
It won't cause a linker error since they don't have the same symbol name. But I will merge it as suggested at the end of the review. :+1: